File: pamoka16/papildomos/uzduotis7.js
window.onload = function() {

    let pradzia = +prompt(`Įveskite intervalo pradžią`);
    let pabaiga = +prompt(`Įveskite intervalo pabaigą`);
    
    teisingasSumosAtsakymas(pradzia, pabaiga)
    
}

function atsitiktinisSveikasSkaicius(pradzia, pabaiga) {
    return Math.round(Math.random() * (pabaiga - pradzia) + pradzia)

}

function teisingasSumosAtsakymas(x, y){

    let a = atsitiktinisSveikasSkaicius(x, y);
    let b = atsitiktinisSveikasSkaicius(x, y);
    let c = a + b
    let ivestis = +prompt(`Įveskite skaičių ${a} ir ${b} sumą`)
    
    while(c != ivestis){
        ivestis = +prompt(`%ved4te neteisingą ${a} ir ${b} sumą. Bandykite iš naujo`)
    }
    alert(`Teisingai. ${a} ir ${b} sumą yra ${a + b}`)
}

File: JAVA/pamoka24-java-loginiai-operatoriai-nested-if/src/uzduotis7.java
public class uzduotis7 {
    public static void main(String[] args) {
        System.out.println("arLygusSkaiciai(15.0247,18.956) = " + arLygusSkaiciai(15.0247,18.956));
        System.out.println("arLygusSkaiciai(12.0327,12.0358) = " + arLygusSkaiciai(12.0327,12.0358));
        System.out.println("arLygusSkaiciai(12.05847,12.05892) = " + arLygusSkaiciai(12.05847,12.05892));
    }

    public static boolean arLygusSkaiciai(double x, double y){
        return Math.round(x * 100d) / 100d == Math.round(y * 100d) / 100d;
    }
}
